xAnalyzer/plugin.cpp

```diff
@@ -119,7 +119,7 @@ PLUG_EXPORT void CBMENUENTRY(CBTYPE cbType, PLUG_CB_MENUENTRY* info)
  			mbp.cbSize = sizeof(MSGBOXPARAMS);
  			mbp.hInstance = pluginHInstance;
  			mbp.lpszCaption = "About...";
- 			mbp.lpszText = "["PLUGIN_NAME " " PLUGIN_VERSION_STR"]\n"
+ 			mbp.lpszText = "[" PLUGIN_NAME " " PLUGIN_VERSION_STR"]\n"
```

Comment:
It seems to not be compiling on VS2015 without space, i guess you could ignore this change if you want to keep solution as VS2013 solution